### ISSUE TITLE
More Jupyter Extensions

### ISSUE THREAD

**Issue by blairdrummond** [StatCan/kubeflow-containers#16]
More extensions that I'll look into

```
s3-browser           
geojson
spreadsheet-viewer  <- this one works, already tested
emacs-keys
jupyter-archive
cpu-status
```

**Comment by blairdrummond** [comment]
jupyterlab-flake8

**Comment by chritter** [comment]
Note that its worth testing the`jupyterlab-flake8` in action, as a installation is not a guarantee for it to work (based on exp).

**Comment by brendangadd** [comment]
@blairdrummond What's our visual Git support looking like right now?

**Comment by blairdrummond** [comment]
Atm, non-existent unless people like `tig`. I think our version of jupyterlab is ahead of the `git` and `flake8` extensions or something, and for some reason it's not working nicely. I'll try to tackle that next.

**Comment by brendangadd** [comment]
@blairdrummond Can you please split out extensions into individual issues so each can have independent:
* Context/motivation
* Hurdles
* Prioritization
* Assignment

**Comment by blairdrummond** [comment]
#21 Will close this for now

**Comment by blairdrummond** [comment]
Closed by #21 . Can reopen a new issue once we need more extensions built-in.
